mod/01_rating/set/type/metric_value.rb

```diff
@@ -190,36 +223,62 @@ def legend args
     subformat(card.metric_card)._render_legend args
   end
 
+  def currency
+    return unless (value_type = Card["#{card.metric_card.name}+value type"])
+    return unless value_type.item_names[0] == 'Monetary' &&
+                  (currency = Card["#{card.metric_card.name}+currency"])
+    currency.content
+  end
+
   view :concise do |args|
-    %{
+    %(
       <span class="metric-year">
         #{card.year} =
       </span>
+      <span class="metric-unit">
+        #{currency}
+      </span>
```

Comment:
This file really  shouldn't know anything about value types.  Could do something like:
`subformat(value_type_card).render_concise_details`

...and then have different detail handling for different metric types.  (?)